src/fragments/lib/logging/ios/setup_logging/20_initialize_with_config.mdx

```diff
@@ -0,0 +1,43 @@
+In your mobile app, create and add a `amplifyconfiguration_logging.json` to the same location as the `amplifyconfiguration.json` file.  Ensure that the file is included in the `Copy Bundle Resources` build phase.
+
+The `<log-group-name>` and `region` is the value you specified in the CDK construct as part of provisioning your AWS backend resources.  The configuration file is the data source for the logging plugin to know where, when and what to send logs.  The example below configures the logging plugin to automatically send all logs at log level `ERROR` at 60 seconds interval and store logs locally up to 1MB.
```

Comment:
The configuration file is the data source for the logging plugin to know where, when and what logs to sends.

Reply:
Updated per suggestion.